fili-core/src/main/java/com/yahoo/bard/webservice/application/AbstractBinderFactory.java

```diff
@@ -208,13 +208,9 @@ protected void configure() {
                 bind(FeatureFlagRegistry.class).to(FeatureFlagRegistry.class);
 
                 // Bard currently expects there to be one web service optimized for low latency queries
-                DruidWebService uiDruidWebService = buildUiDruidWebService(getMappers().getMapper());
+                DruidWebService druidWebService = buildDruidWebService(getMappers().getMapper());
 
-                // As well as one for open ended, potentially long running queries
-                DruidWebService nonUiDruidWebService = buildNonUiDruidWebService(getMappers().getMapper());
-
-                bind(uiDruidWebService).named("uiDruidWebService").to(DruidWebService.class);
-                bind(nonUiDruidWebService).named("nonUiDruidWebService").to(DruidWebService.class);
+                bind(druidWebService).named("druidWebService").to(DruidWebService.class);
```

Comment:
We should be able to remove the call to `.named("druidWebService")` since there's only one. Then you can remove the annotations that have `@Named("druidWebService")`